src/commands/service/pipeline.ts

```diff
@@ -189,6 +192,14 @@ export const execute = async (
     }
 
     await fetchValues(serviceName, opts);
+    const bedrockFile: BedrockFile = loadBedrockFile(projectPath);
+    let servicePath = "";
+    bedrockFile.services.forEach((service) => {
+      if (service.displayName === serviceName) {
+        servicePath = service.path;
+        return;
+      }
+    });
```

Comment:
if serviceName does not always found in services array then we need to deal with it :-)

```
const found = bedrockFile.services.find(s => s. displayName === serviceName);
if (!found) {
  throw ...
}
const servicePath = found.path;
```